app/containers/message/Components/Attachments/ImageGallery/ImageGalleryItem.tsx

```
@@ -0,0 +1,43 @@
+	height: number;
+	overflowCount?: number;
+}
+
+const ImageGalleryItem = ({ file, author, showAttachment, width, height, overflowCount = 0 }: IImageGalleryItem) => {
+	const { colors } = useTheme();
+	const { status, onPress, url, isEncrypted } = useMediaAutoDownload({ file, author, showAttachment });
+
+	return (
+		<Touchable onPress={onPress} style={[styles.cell, { width, height }]}>
+			<WidthAwareContext.Provider value={width}>
+				<MessageImage uri={url} status={status} encrypted={isEncrypted} imagePreview={file.image_preview} imageType={file.image_type} />
```

ESLint

[error] Replace `·uri={url}·status={status}·encrypted={isEncrypted}·imagePreview={file.image_preview}·imageType={file.image_type}·` with `⏎↹↹↹↹↹uri={url}⏎↹↹↹↹↹status={status}⏎↹↹↹↹↹encrypted={isEncrypted}⏎↹↹↹↹↹imagePreview={file.image_preview}⏎↹↹↹↹↹imageType={file.image_type}⏎↹↹↹↹`

(prettier/prettier)